action/recipes/linux_test.go

```diff
@@ -0,0 +1,169 @@
+// SPDX-License-Identifier: MIT
+
+// Package recipes / linux
+package recipes
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"dagger.io/dagger"
+	"github.com/9elements/firmware-action/action/container"
+	"github.com/9elements/firmware-action/action/filesystem"
+	"github.com/Masterminds/semver"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLinux(t *testing.T) {
+	// This test is really slow (like 100 seconds)
+	/*
+		if testing.Short() {
+			t.Skip("skipping test in short mode")
```

Comment:
Any particular reason to comment out the skip in short mode?

Reply:
Oversight, my bad.